File: Vuejs+OOP/Player.js
/** @typedef {import('./CardDeck').Card} Card */

export default class Player {
  /** @type {Card[]} */
  cards = [];

  /** @type {number} */
  #point = 0

  init() {
    this.cards = [];
    this.#point = 0;
  }

  /**
   * @param {Card} card
   */
  addCard(card) {
    this.cards.push(card);
    this.#point += card.val;
  }

  get point () {
    return this.#point;
  }
}

File: Go/player/player.go
package player

import (
	"fmt"

	card "../card"
)

type Player struct {
	cards []*card.Card
	Name  string
	Score byte
}

func (p *Player) AddCard(card *card.Card) {
	p.Score += card.Value
	p.cards = append(p.cards, card)
}

func (p *Player) Display(mask bool) {
	if mask {
		fmt.Printf("%s %d? %s [**]\n", p.Name, p.cards[0].Value, p.cards[0])
	} else {
		cardStr := ""

		for _, card := range p.cards {
			cardStr += fmt.Sprintf("%s ", card)
		}

		fmt.Printf("%s %d %s\n", p.Name, p.Score, cardStr)
	}
}
